examples/secure_token_storage.py

```diff
@@ -2,13 +2,15 @@
 # You should load the token as an environment variable, then access it like this:
 
 import os
+from nextcord.ext import commands
 
 # There are other ways to load environment variables but this is one
 from dotenv import load_dotenv # https://pypi.org/project/python-dotenv/
 
 # load_dotenv reads from a file called .env in the same directory as the python files which should roughly look like TOKEN="1234567890"
 load_dotenv()
 token = os.getenv('BOT_TOKEN')
+bot = commands.Bot()
```

Comment:
`commands.Bot()` does not have a default value for `command_prefix` so I think we should not be leaving that out in the example, unless something like #101 gets merged.

Reply:
That won't be merged as vco had a pretty good point, maybe making the prefix something simple like ! Or ?

Reply:
Yeah I agree we need a example prefix here. Maybe use the same as in the other examples?

Reply:
I would suggest "?" as it is the prefix of all other examples

Reply:
> I would suggest "?" as it is the prefix of all other examples

We have been using "$" everywhere else.

Reply:
Oh whoops, I meant $, lol.